ui/src/message_popup/hint_card.jsx

```diff
@@ -0,0 +1,141 @@
+// @flow
+import React from 'react';
+import * as PropTypes from '../prop_types.js';
+import RaisedButton from 'material-ui/RaisedButton';
+import Divider from 'material-ui/Divider';
+
+/*
+Shows a card for a student from the virtual school.
```

Comment:
Stale comment here, can you update?  This would be a good place to put your comment in the PR about "this shows a hint and cycles through examples and non-examples"